### PR TITLE
Feature/display 1028 login page

### PR DESCRIPTION
**Pull request by sinejespersen**
#### Link to ticket

https://jira.itkdev.dk/browse/DISPLAY-1028
https://jira.itkdev.dk/browse/DISPLAY-1027

#### Description

- update login page (see image below)
- add todo for button with mitid
- clean up translations a bit
- fix some WCAG2.1 issues
- add os2display logo

#### Screenshot of the result

image

image

#### Checklist

- [ ] My code is covered by test cases.
- [ ] My code passes our test (all our tests).
- [ ] My code passes our static analysis suite.
- [ ] My code passes our continuous integration process.

If your code does not pass all the requirements on the checklist you have to add a comment explaining why this change 
should be exempt from the list.

#### Additional comments or questions

If you have any further comments or questions for the reviewer please add them here.

### CODE REVIEW

File: src/components/user/aarhus-logo.svg
Comment (tuj) [inline src/components/user/aarhus-logo.svg:1]: General instead of Aarhus specific

File: src/components/user/login.jsx [hunk 1]
```diff
@@ -13,7 +13,9 @@ import { api } from "../../redux/api/api.generated";
 import ConfigLoader from "../../config-loader";
 import { displayError } from "../util/list/toast-component/display-toast";
 import localStorageKeys from "../util/local-storage-keys";
-
+import LoginSidebar from "../navigation/login-sidebar/login-sidebar";
+import AarhusLogo from "./aarhus-logo.svg";
```
Comment (tuj) [inline src/components/user/login.jsx:17]: General instead of Aarhus specific

File: src/translations/da/common.json [hunk 2]
```diff
@@ -27,6 +20,10 @@
       "load-theme-error": "Der skete en fejl da temaet med følgende id: {{id}} skulle hentes:"
     }
   },
+  "login-sidebar": {
+    "AD-info-text": "Er du <strong>medarbejder</strong> i skal du benytte AD login.",
```
Comment (tuj) [inline src/translations/da/common.json:24]: "Er du <strong>medarbejder</strong> i skal du benytte AD login.",
=>
"Er du <strong>medarbejder</strong> skal du benytte AD login.",

**Review by tuj — CHANGES_REQUESTED** [review]

File: src/translations/da/common.json (hunk 2 above)
Comment (sinejespersen) [reply src/translations/da/common.json:24]: Ja sikke da noget vås

File: src/components/user/login.jsx (hunk 1 above)
Comment (sinejespersen) [reply src/components/user/login.jsx:17]: image

**Review by tuj — APPROVED** [review]

**Review by yepzdk — APPROVED** [review]

**Pull request merged**